app/helpers/sessions_helper.rb

```diff
@@ -4,15 +4,36 @@ def log_in user
   end
 
   def current_user
-    @current_user ||= User.find_by(id: session[:user_id])
+    if (user_id = session[:user_id])
+      @current_user ||= User.find_by id: user_id
+    elsif (user_id = cookies.signed[:user_id])
+      user = User.find_by id: user_id
+      if user && user.authenticated?(cookies[:remember_token])
+        log_in user
+        @current_user = user
+      end
+    end
   end
 
   def logged_in?
     !current_user.nil?
```

Comment:
sử dụng present? thay cho nil thử em nhé

Reply:
khi em sử dụng present? thì rubocop báo lỗi:

Use current_user.present? instead of !current_user.blank?.
    !current_user.blank?
    ^^^^^^^^^^^^^^^^^^^^